### PR TITLE
Login with OTP

### PR DESCRIPTION
**Pull request by ks-rahul**
1. Login with OTP api 
2. Not verified user verify when login API 
3. Tab name/title should be "Canonizer" instead of "Default Meta tags for canonizer"
4. form validation change to onBlur from onChange
5. registration form middle name remove

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,22 @@
 REACT_APP_TIMEOUT=1000
```
Comment (akuks) [inline .env]: This files needs to be deleted. or shouldn't be pushed.

Comment (akuks) [inline .env]: This files needs to be deleted. or shouldn't be pushed.

Comment (akuks) [reply .env]: if you have any conf updates. Please update the .env.example file

Comment (akuks) [reply .env]: if you have any conf updates. Please update the .env.example file

File: src/components/ComponentPages/Login/UI/index.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ import styles from "./Login.module.scss";
 import messages from "../../../../messages";
 import SocialLoginButton from "../../../common/socialLogin";
 import FormItem from "../../../common/formElements";
+// import { formValidationTypes } from "src/messages/validationRules";
```
Comment (akuks) [inline src/components/ComponentPages/Login/UI/index.tsx]: Can we remove this?

Comment (akuks) [inline src/components/ComponentPages/Login/UI/index.tsx]: Can we remove this?

File: yarn.lock [hunk 3]
```diff
@@ -36,27 +36,20 @@
...
-
-"@babel/code-frame@^7.12.13":
-  "integrity" "sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg=="
-  "resolved" "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.16.7.tgz"
-  "version" "7.16.7"
-  dependencies:
-    "@babel/highlight" "^7.16.7"
-
-"@babel/code-frame@^7.16.0":
+"@babel/code-frame@^7.10.4", "@babel/code-frame@^7.12.13", "@babel/code-frame@^7.16.0":
```
Comment (akuks) [inline yarn.lock]: Please remove this file

Comment (akuks) [inline yarn.lock]: Please remove this file

**Review by akuks — CHANGES_REQUESTED** [review]
please do the necessary changes

File: .env (hunk 1 above)
Comment (ks-rahul) [reply .env]: removed

File: src/components/ComponentPages/Login/UI/index.tsx (hunk 2 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Login/UI/index.tsx]: line removed

Comment (ks-rahul) [reply src/components/ComponentPages/Login/UI/index.tsx]: line removed

File: yarn.lock (hunk 3 above)
Comment (ks-rahul) [reply yarn.lock]: file removed

Comment (ks-rahul) [reply yarn.lock]: file removed

**Review by akuks — APPROVED** [review]

**Pull request merged**